### ISSUE TITLE
Outdated copyright year

### ISSUE THREAD

**Issue by jfushada** [yunity/karrot-frontend#997]
Hello! In the LICENSE file, it says that "Copyright (c) 2017 Foodsaving Worldwide Dev Team". Should it be updated to 2018?

**Comment by tiltec** [comment]
Indeed, it should be updated to 2018 because we made relevant changes this year: https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year

I wonder if there's a bot around that can take care of this automatically...

**Comment by peterkshultz** [comment]
React found a workaround to this problem:

`Copyright (c) 2013-present, Facebook, Inc.`

https://github.com/facebook/react/blob/master/LICENSE

**Comment by tiltec** [comment]
Seems a good solution and a quick fix :)

**Comment by PhrydRhys** [comment]
Hello,
I would like to work on this issue.
Would the appropriate timeframe for the license be
> Copyright (c) 2017-Present Foodsaving Worldwide Dev Team

now?

**Comment by tiltec** [comment]
Thanks! Actually, it should say "2016-present", as the frontend history goes back to 2016.
